src/avram/migrator/runner.cr

```diff
@@ -48,33 +48,35 @@ class Avram::Migrator::Runner
   end
 
   def self.drop_db(quiet? : Bool = false)
-    run "dropdb #{cmd_args}"
+    DB.connect(credentials.connection_string) do |db|
+      db.exec "DROP DATABASE IF EXISTS #{db_name}"
+    end
     unless quiet?
       puts "Done dropping #{Avram::Migrator::Runner.db_name.colorize(:green)}"
     end
-  rescue e : Exception
-    if (message = e.message) && message.includes?(%("#{self.db_name}" does not exist))
-      unless quiet?
-        puts "Already dropped #{self.db_name.colorize(:green)}"
-      end
-    else
-      raise e
-    end
   end
 
   def self.create_db(quiet? : Bool = false)
-    run "createdb #{cmd_args}"
+    DB.connect(credentials.connection_string) do |db|
+      db.exec "CREATE DATABASE #{db_name} WITH OWNER DEFAULT"
+    end
     unless quiet?
-      puts "Done creating #{Avram::Migrator::Runner.db_name.colorize(:green)}"
+      puts "Done creating #{db_name.colorize(:green)}"
+    end
+  rescue e : DB::ConnectionRefused
```

Comment:
Not sure what level the issue is at, but when you can't connect, a `PQ::PQError` is raised with a nice helpful error... then that gets swallowed somewhere and a `DB::ConnectionRefused` is raised in it's place with no error message... so `e.message` is nil. They just expect you to read through the stack trace.